src/main/java/org/cyclops/integrateddynamics/core/evaluate/operator/Operators.java

```diff
@@ -1182,6 +1174,20 @@ public IValue evaluate(OperatorBase.SafeVariablesGetter variables) throws Evalua
                 return ValueTypeList.ValueList.ofFactory(new ValueTypeListProxySlice<>(list, from.getRawValue(), to.getRawValue()));
             }).build());
 
+    public static final IOperator LIST_INTERSECTION = REGISTRY.register(OperatorBuilders.LIST
+            .inputTypes(ValueTypes.LIST, ValueTypes.LIST)
+            .renderPattern(IConfigRenderPattern.PREFIX_2_LONG).output(ValueTypes.LIST)
+            .symbol("∩").operatorName("intersection")
+            .function(variables -> {
+                IValueTypeListProxy<IValueType<IValue>, IValue> rawList1 = variables.getValue(0, ValueTypes.LIST).getRawValue();
+                IValueTypeListProxy<IValueType<IValue>, IValue> rawList2 = variables.getValue(1, ValueTypes.LIST).getRawValue();
+
+                LinkedHashSet<IValue> result = Sets.newLinkedHashSet(rawList1);
+                result.retainAll(Sets.newLinkedHashSet(rawList2));
+
```

Comment:
Just one more thing I just thought of: we need to error if one of the lists is infinite.
Otherwise, the server may crash.

Similar to this: https://github.com/CyclopsMC/IntegratedDynamics/blob/daac9ddde2364c0068807afad36ca86ac0b5c9ee/src/main/java/org/cyclops/integrateddynamics/core/evaluate/operator/Operators.java#L979-L982

Reply:
I'm a little stumped by this;
![image](https://user-images.githubusercontent.com/286341/195058581-01e7f9a3-81be-4331-9755-2e7bf1693eef.png)
I can't tell what I'm doing differently to your example which doesn't raise this error...